ocpi-tariffs/src/types/time.rs

```diff
@@ -65,13 +65,16 @@ impl From<Duration> for HoursDecimal {
 
 impl AddAssign for HoursDecimal {
     fn add_assign(&mut self, rhs: Self) {
-        self.0 = self.0 + rhs.0;
+        self.0 = self
```

Comment:
You're implementing a `saturating_add` without calling it so.
Consider ditching the `AddAssign` impl and defining a `saturating_add` instead